core/common/src/test/java/alluxio/security/authorization/PermissionTest.java

```diff
@@ -124,4 +124,57 @@ private void verifyPermission(String user, String group, short mode, Permission
     Assert.assertEquals(group, permission.getGroup());
     Assert.assertEquals(mode, permission.getMode().toShort());
   }
+
+  /**
+   * Tests the {@link Permission#getOwnerBits(short)} method.
+   */
+  @Test
+  public void getOwnerBitsTest() throws Exception {
+    final short[] inputs = new short[] {
+        0700,
+        0505,
+        0640,
+        0000,
+    };
+    final short[] results = new short[] {
+        0007,
+        0005,
+        0006,
+        0000,
+    };
+    for (int i = 0; i < inputs.length; i++) {
+      Assert.assertEquals(results[i], Permission.getOwnerBits(inputs[i]));
```

Comment:
I think this would be easier to understand as 
```java
@Test
public void getOwnerBitsTest() throws Exception {
  Assert.assertEquals(0007, Permission.getOwnerBits(0700));
  Assert.assertEquals(0005, Permission.getOwnerBits(0505));
  Assert.assertEquals(0006, Permission.getOwnerBits(0640));
  Assert.assertEquals(0000, Permission.getOwnerBits(0000));
}
```

Same for the test below